app/Http/Controllers/AuthenticateController.php

```diff
@@ -32,6 +32,17 @@ public function authenticate(Request $request){
                    ]);
   }  
 
+  //refresh route for the front end
+  public function user(){
+    $user = JWTAuth::parseToken()->toUser();
+    $token = JWTAuth::parseToken()->refresh(); 
+    $data = array();
+    $data["user"] = $user;
+    return response($data)->withHeaders([
+                      'Authorization'=> 'Bearer: '.$token
```

Comment:
Are we sending this data back to the client asking for their permission for their data? Why not embed the token in the body of the response?